docs/application/dotnet/guides/nui/tableview.md

```diff
@@ -1,40 +1,52 @@
 # TableView
 
-The `TableView` class is a layout container for aligning child actors in a grid like layout.
-`TableView` constraints the x and y position, width, and height of the child actors.
-The Z position and depth are left intact, so that the three-dimensional model actors can also be laid out in a grid without losing their depth scaling.
+The [TableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TableView.html) class is a layout container for aligning child actors in a grid like layout.
+Based on `TableView` and child's size, child's X and Y position are defined. 
+
+In case of width sum of children in row is lower than width of `TableView`, children are positioned with equal padding then. Analogous rule are used when sum of children height in column is lower than height of `TableView`:
```

Comment:
Suggesting a rephrase, please verify if the technical meaning is accurate as you wanted:

In case the sum of children's width in a row is lower than the width of `TableView`, then the children are positioned with equal padding. Analogous rule are used when the sum of children's height in a column is lower than the height of `TableView`.